### ISSUE TITLE
Implement Women's Health Text Reminders. 

### ISSUE THREAD

**Issue by shanemelliott** [department-of-veterans-affairs/va.gov-team#24821]
From: Ashe, Steven B <Steven.Ashe2@va.gov> 
Sent: Friday, May 14, 2021 12:52 PM
To: Williams, Erica <Erica.Williams@va.gov>
Cc: VEText <VEText@va.gov>
Subject: RE: Women's Vet Outreach via Vet-text
Importance: High

Good Friday Erica,

Attached is the list of female veterans who will qualify for the Women’s Screening Text Message Reminder, which will include the following message:

“According to our records, you are due for your cervical cancer screening (or mammogram if that is the case).  Please contact your primary care team through the call center at 562-826-5151.  If you have already completed your screening please notify your primary care team who can document the screening in your medical record."

Please send a screenshot of what this text message will look like so I can share it with the Clinicians.  Let me know if you have any questions.

**Comment by shanemelliott** [comment]
[https://app.zenhub.com/files/133843125/3a6910ce-1b81-466a-ac01-052b3cd4ddfe/download](https://app.zenhub.com/files/133843125/3a6910ce-1b81-466a-ac01-052b3cd4ddfe/download)

**Comment by jwowk** [comment]
OVAC Approval Form (https://app.zenhub.com/files/133843125/7109c3d8-90ed-46b3-b0e8-126aa2b8a3cd/download)

**Comment by jwowk** [comment]
Karen Hallum from Tuscaloosa VA reached out regarding Women's Health reminders as well. 6/24/21 Shane introduced her to Steven Ashe and Laurie Rudisill (original requesters) to combine efforts.

**Comment by kjduensing** [comment]
Approved by OVAC, but still waiting on info from site

**Comment by kjduensing** [comment]
waiting on the site to come back with time, date, message. Last checked - reviewing list of users (had found some inconsistencies).

**Comment by Kennon-git** [comment]
approved, but not ready to send messages yet. when ready we set them up for self service.

**Comment by pcshah1026** [comment]
on hold.  pending VHA/OGC privacy review

**Comment by jwowk** [comment]
Pilot went live with 523 for mammography screening
